OpenCL_C.txt

```diff
@@ -307,7 +307,8 @@ The following table describes the list of built-in scalar data types.
       The `double` data type must conform to the IEEE 754 double precision
       storage format.
 
-      <<unified-spec, Requires>> support for OpenCL C 1.2 or newer.
+      <<unified-spec, Requires>> support for OpenCL C 1.2 or newer. In
+      OpenCL C 3.0 it requires support of {opencl_c_fp64} feature.
```

Comment:
```suggestion
      OpenCL C 3.0 it requires support of the {opencl_c_fp64} feature.
```
Elsewhere it looks like we refer to the feature macro rather than the feature. I'm not sure if there is any distinction in meaning?

Reply:
Ah I think it was done when we didn't have wording saying that features inherit names from the macros but now we do. For extensions we used to refer to the name rather than the macro. This is why we wanted to update the features but it has not been done yet.

I can reword it to alight with the other features and we change all of them later or I can keep it as is since this is what we aim at but it will be inconsistent with the other features but consistent with extensions...

Reply:
OK thanks, however you want to stage it is fine with me

Reply:
I do think we should accept this suggestion though and add `the`.